pkg/kubectl/cmd/attach/attach.go

```diff
@@ -189,7 +189,7 @@ func (o *AttachOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []s
 		fullCmdName = cmdParent.CommandPath()
 	}
 	if len(fullCmdName) > 0 && cmdutil.IsSiblingCommandExists(cmd, "describe") {
-		o.SuggestedCmdUsage = fmt.Sprintf("Use '%s describe pod/%s -n %s' to see all of the containers in this pod.", fullCmdName, o.PodName, o.Namespace)
+		o.SuggestedCmdUsage = fmt.Sprintf("Use '%s describe pod/%%s -n %s' to see all of the containers in this pod.", fullCmdName, o.Namespace)
```

Comment:
using SuggestedCmdUsage as a format string means that if fullCmdName or o.Namespace contain `%` characters, the output will be malformed

Reply:
Thanks for pointing this out. I checked Namespace is only allowed to be `[a-z0-9]([-a-z0-9]*[a-z0-9])?`, but can't be sure that fullCmdName won't contain `%`, it should be kubelet for most cases though. 
To avoid potential problem, I move the suggested usage formatting to where it could be done completely. The issue in exec is also addressed.
PTAL. Thanks!